README.md

```diff
@@ -43,11 +33,14 @@ export WERKZEUG_DEBUG_PIN=off
 "> environment.sh
 ```
 
-NOTES:
+Things to change:
 
- * Replace the placeholder key and prefix values as appropriate
- * The SECRET_KEY and DANGEROUS_SALT should match those in the [notifications-admin](https://github.com/alphagov/notifications-admin) app.
- * The  unique prefix for the queue names prevents clashing with others' queues in shared amazon environment and enables filtering by queue name in the SQS interface.
+* Replace `YOUR_OWN_PREFIX` with `local_dev_<first name>`.
+* Run the following in the credentials repo to get the API keys.
+
+```
+notify-pass credentials/providers/api_keys
+```
 
 ### Postgres
```

Comment:
Can we update the Postgres part we don't need to down grade to version 9.6.

Reply:
I remember trying this app with v13, but some of the tests fail as we look for specific strings in the exceptions. We could say v9.X, or maybe point to the infra version setting. How about that?

Reply:
It should be at least version 11

Reply:
So it is. I've added another commit to update the instructions there. Good to know we're nearer the cutting edge 👍.

Reply:
i'd remove the "up to", so rather than "works with versions up to v11" i'd say "works with version v11" or something. we dont want to encourage people to install pg versions that don't match prod.

Reply:
Sure, I've amended the commit with that 👍.